string robot
string location
string destination
string floor
---
string result